packages/block-library/src/site-logo/index.php

```diff
@@ -40,7 +40,7 @@ function render_block_core_site_logo( $attributes ) {
 		// Add the link target after the rel="home".
 		// Add an aria-label for informing that the page opens in a new tab.
 		$aria_label  = 'aria-label="' . esc_attr__( '(Home link, opens in a new tab)' ) . '"';
-		$custom_logo = str_replace( 'rel="home"', 'rel="home" target="' . $attributes['linkTarget'] . '"' . $aria_label, $custom_logo );
+		$custom_logo = str_replace( 'rel="home"', 'rel="home" target="' . esc_attr( $attributes['linkTarget'] ) . '"' . $aria_label, $custom_logo );
```

Comment:
This is under a `'_blank' === $attributes['linkTarget']` check, so we already know what the value is, and it's hardcoded without any user-input involved. Escaping this string is not necessary, it can just be `_blank`

Reply:
IMHO, if data is coming from a variable I believe we should escape at point of output. Who's to say what the variable might contain in future. It's hardcoded for now but that's no guarantee it will stay that way.

This is minor overhead for the sake of long term peace of mind and resilience against XSS.

Reply:
I agree... But why not just make it `target="_blank"`? It can't be anything else. That way we make it easier to read, and at the same time it's a performance micro-optimization 🤔

Reply:
Let me look again. If we can hard code inline when we build the markup then let's do that,

Reply:
Ok I've looked again. Whilst I recognise that this whole block is currently behind an explicit conditional that could easily change in the future. 

What if there's a refactor and the `$custom_logo` markup ends up outside this conditional. What if the developer forgets to escape the `linkTarget`?

I appreciate this is a "what if" scenario but given the low overhead (how many site logo blocks are going to be on one page?) I believe it is better to be safe than sorry when it comes to XSS. I believe it's a good trade off.

Reply:
What's the next step here? Is the addition of this `esc_attr()` a blocker? If it absolutely is I will revert the change.